src/assets/scripts/client/base/Compass.js

```diff
@@ -0,0 +1,57 @@
+import { radians_normalize } from '../math/circle';
+
+/**
+ * @class Compass
+ */
+class Compass {
+    /**
+     * @constructor
+     */
+    // istanbul ignore next
+    constructor() {
+        /**
+         * Magnetic declination, in radians east
+         *
+         * @property _magneticNorth
+         * @type {number}
+         * @default 0
```

Comment:
`@for Compass`

Reply:
not for properties, only methods/functions